docker/provisioner/connectors/Pipfile

```diff
@@ -6,6 +6,8 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
+cfnresponse = "*"
+requests = "*"
```

Comment:
please add versions

Reply:
It is not needed to add version here, because the used versions bumped in the lock file.